packages/node-build-scripts/src/sass/sassCompileFile.mjs

```diff
@@ -53,5 +50,5 @@ function fixSourcePathsInSourceMap({ outputMapFile, rawSourceMap }) {
         const pathToSourceWithoutProtocol = source.replace("file://", "");
         return relative(outputDirectory, pathToSourceWithoutProtocol);
     });
-    return new SourceMapGenerator(rawSourceMap).toString();
+    return SourceMapGenerator.fromSourceMap(new SourceMapConsumer(rawSourceMap)).toString();
```

Comment:
it looks like we were doing this wrong, because I was getting empty source maps. this new approach seems to work better.

docs: https://github.com/7rulnik/source-map-js#generating-a-source-map